### PR TITLE
Added autoscaler example with schedule

### PR DESCRIPTION
**Pull request by betsy-lichtenberg**
Planning to use on this page:

https://cloud.google.com/compute/docs/autoscaler/scaling-schedules#creating_a_scaling_schedule

### CODE REVIEW

**Comment by comment-bot-dev** [comment]
@betsy-lichtenberg
Thanks for the PR! 🚀
Unfortunately it looks like some of our CI checks failed. See the [Contributing Guide](../blob/master/CONTRIBUTING.md) for details.
- ⚠️check_headers
 All files need a license header. Please make sure all your files include the appropriate header. A helper tool available [here](https://github.com/google/addlicense). 
 ```bash 
Checking file headers
./autoscaler_instance_group.tf
 ``` 
- ⚠️check_terraform
 Failed Terraform check. More details below. 
 ```bash 
Running terraform fmt
autoscaler_instance_group.tf
--- old/autoscaler_instance_group.tf
+++ new/autoscaler_instance_group.tf
@@ -17,12 +17,12 @@
     }
 
     scaling_schedules {
-      name                   = "every-weekday-morning"
-      description            = "Increase to 2 every weekday at 7AM for 12 hours."
-      min_required_replicas  = 2
-      schedule               = "0 7 * * MON-FRI"
-      time_zone              = "America/New_York"
-      duration_sec           = 43200
+      name                  = "every-weekday-morning"
+      description           = "Increase to 2 every weekday at 7AM for 12 hours."
+      min_required_replicas = 2
+      schedule              = "0 7 * * MON-FRI"
+      time_zone             = "America/New_York"
+      duration_sec          = 43200
     }
   }
 }
Error: terraform fmt failed with exit code 3
Check the output for diffs and correct using terraform fmt <dir>
 ```

**Comment by comment-bot-dev** [comment]
@betsy-lichtenberg
Thanks for the PR! 🚀
Unfortunately it looks like some of our CI checks failed. See the [Contributing Guide](../blob/master/CONTRIBUTING.md) for details.
- ⚠️check_terraform
 Failed Terraform check. More details below. 
 ```bash 
Running terraform fmt
autoscaler_instance_group.tf
--- old/autoscaler_instance_group.tf
+++ new/autoscaler_instance_group.tf
@@ -33,12 +33,12 @@
     }
 
     scaling_schedules {
-      name                   = "every-weekday-morning"
-      description            = "Increase to 2 every weekday at 7AM for 12 hours."
-      min_required_replicas  = 2
-      schedule               = "0 7 * * MON-FRI"
-      time_zone              = "America/New_York"
-      duration_sec           = 43200
+      name                  = "every-weekday-morning"
+      description           = "Increase to 2 every weekday at 7AM for 12 hours."
+      min_required_replicas = 2
+      schedule              = "0 7 * * MON-FRI"
+      time_zone             = "America/New_York"
+      duration_sec          = 43200
     }
   }
 }
Error: terraform fmt failed with exit code 3
Check the output for diffs and correct using terraform fmt <dir>
 ```

**Comment by comment-bot-dev** [comment]
@betsy-lichtenberg
Thanks for the PR! 🚀<br/>✅ Lint checks have passed.

File: autoscaler_instance_group/main.tf [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+  version {
+    instance_template = google_compute_instance_template.default.id
+    name              = "primary"
+  }
+
+  base_instance_name = "autoscaler-sample"
+}
+
+data "google_compute_image" "debian_9" {
```
Comment (ahwilms) [inline autoscaler_instance_group/main.tf:85]: since the family is debian 11, should the name for this image be debian_11?

**Review by ahwilms — APPROVED** [review]
Small comment, otherwise looks good.

File: autoscaler_instance_group/main.tf (hunk 1 above)
Comment (betsy-lichtenberg) [reply autoscaler_instance_group/main.tf]: Good catch!

**Comment by comment-bot-dev** [comment]
@betsy-lichtenberg
Thanks for the PR! 🚀<br/>✅ Lint checks have passed.

**Pull request merged**